gateway/src/apicast/policy/load_configuration/load_configuration.lua

```diff
@@ -30,4 +31,8 @@ function _M:rewrite(context)
     context.configuration = configuration_loader.rewrite(self.configuration, context.host)
 end
 
+function _M.ssl_certificate(_, context)
+    context.host = ssl.server_name()
```

Comment:
`ssl.server_name()` can return an error: https://github.com/openresty/lua-resty-core/blob/master/lib/ngx/ssl.md#server_name
Shouldn't we handle it?

Reply:
If it returns error then `context.host` is going to be nil and no service policy chain will be found.
In that case we serve whatever certificate was set to APICAST_HTTPS_CERTIFICATE or the provided default. 

Maybe we should just log the error with some low log level like debug.